### DISCUSSION TITLE
Authorization .net 5

### DISCUSSION THREAD

**Discussion by 420skywalker420** [serenity-is/Serenity#5612] [Help]
Hi,

Can someone show how to get  current logged user? And how to chek his permissions, because this doesn't work?

var user = (UserDefinition)Authorization.UserDefinition;
            if (!Authorization.HasPermission(PermissionKeys.Tenants))
                query.Where(fldTenantId == user.TenantId);

Thank you!

**Comment by minhhungit** [comment]
Use IUserAccessor, for example:

```csharp
public class DemoWorld{
    IUserAccessor _userAccessor;
    
    public DemoWorld(IUserAccessor userAccessor) // this is your class constructor
    {
        _userAccessor = userAccessor;
    }
}
````

Then use `_userAccessor `in somewhere like this: `_userAccessor.User....`
